### ISSUE TITLE
🐛 [BUG] - TYPING DATE IN SOME SPECIFIC FORMAT NOT WORKING

### ISSUE THREAD

**Issue by haseebmanzoorcbt** [walkframe/gridsheet#78]
### Description

when adding date in some specific cell in gridsheet, i get wrong date. I tried using onChange but it already overwrites new wrong value before getting it 

i-e : when typing 22 November 2024 , it sets to 2200-01-01

### Reproduction URL

https://github/haseebcbt

### Reproduction steps

```bash
1. Go to Grid sheet
2. Click on some cell
3. Type date like 22 Nov 2024
4. Output is 2200-01-01 on changing/selecting other cell
```

### Version

^1.1.1-alpha.0

### Browsers

Firefox, Chrome, Safari, Microsoft Edge, Opera

### OS

Windows, Linux, Mac

**Comment by righ** [comment]
Thank you for the report.
It seems the bug is caused by the date-fns-timezone parser not supporting the format string you entered.
Please wait a moment while I investigate whether I can replace it with another library.

**Comment by haseebmanzoorcbt** [comment]
Sure no issues !

**Comment by righ** [comment]
Sorry, I unintentionally closed the Issue. Please Reopen if you cannot confirm the fix.
I have fixed this issue in the latest version, [1.3.0-alpha.2](https://www.npmjs.com/package/@gridsheet/react-core/v/1.3.0-alpha.3).
Also, you can create [your own parser](https://docs.walkframe.com/gridsheet/react#optionsparsers) to parse date/time as you like, so if you want to support other formats, please consider using this one as well.
